### PR TITLE
fix-swift-testbed-crash-DEVEX-327

### PR DESCRIPTION
**Pull request by E-B-Smith**
https://branch.atlassian.net/browse/DEVEX-327

https://fabric.io/branch-metrics/ios/apps/io.branch.testbed-swift/issues/5acd261736c7b23527213313?time=1523318400000%3A1523404799000

### CODE REVIEW

**Comment by E-B-Smith** [comment]
@sojanpr

**Comment by BranchMacMini** [comment]
Result of [Integration 1](https://stlt.herokuapp.com/v1/xcs_deeplink/qabot.stage.branch.io/d336a704184380e6c3299249226a6afa/d336a704184380e6c3299249226a6d1c)
---
*Duration*: 27 minutes and 45 seconds
*Result*: **Build failed 2 tests** out of 227

File: Branch-TestBed-Swift/TestBed-Swift/AppDelegate.swift [hunk 1]
```diff
@@ -98,14 +101,20 @@ class AppDelegate: UIResponder, UIApplicationDelegate, AdjustDelegate, AppsFlyer
                     if clickedBranchLink {
                         let nc = self.window!.rootViewController as! UINavigationController
                         let storyboard = UIStoryboard(name: "Content", bundle: nil)
-                        let contentViewController = storyboard.instantiateViewController(withIdentifier: "Content") as! ContentViewController
+                        let contentViewController = storyboard.instantiateViewController(
+                            withIdentifier: "Content"
+                        ) as! ContentViewController
                         nc.pushViewController(contentViewController, animated: true)
                         
-                        let referringLink = paramsDictionary["~referring_link"] as! String
```
Comment (sojanpr) [inline Branch-TestBed-Swift/TestBed-Swift/AppDelegate.swift]: Was this line causing the crash? Is there chance we get  anything other than type String for `referring link`?

Comment (E-B-Smith) [reply Branch-TestBed-Swift/TestBed-Swift/AppDelegate.swift]: It was mis-cast.

**Comment by sojanpr** [comment]
LGTM 👍

**Pull request merged**